lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -884,6 +891,7 @@ def _get_default_parameters(kymo, channel):
             r"When tracking, the algorithm searches for points in future frames to connect. Points "
             r"within a certain distance from the expected future position are connected. The "
             r"diffusion parameter determines how quickly this distance grows over time.",
+            display_unit=f"{kymo._calibration.unit_label}²/s",
```

Comment:
One thing that strikes me as odd is that `unit_label` is TeX style, while here we have a unicode squared symbol. Not really an issue, but curious.

That said, do we want this label as TeX?

Reply:
Either the whole unit becomes a $..$ math section, or none of it, IMO.

Reply:
I agree. Personally, I'm leaning towards not having the unit be a math section, but just having the mu symbol in `Kymo`'s unit label.

Reply:
I've decided to leave that issue for a separate PR, OK?

Reply:
Sure.